lib/runnable-hostname.js

```diff
@@ -0,0 +1,65 @@
+'use strict';
+
+var exists = require('101/exists');
+var createHostLabel = require('./create-host-label.js');
+
+module.exports = {
+  elastic: elastic,
+  direct:  direct
+};
+
+function elastic (opts) {
+  validateOpts(opts);
+  // consistency btw direct and elastic
+  return direct(opts).replace(opts.shortHash+'-', '');
+}
+
+function direct (opts) {
+  validateOpts(opts);
+  var template = '{shortHash}-{repoName}-{env}-{ownerUsername}'
+    .replace('{shortHash}', opts.shortHash)
+    .replace('{env}', 'staging');
```

Comment:
I didn't toLowerCase it bc I want to frontend to maintain how it currently displays urls with the proper casing